### DISCUSSION TITLE
Splash screen duration

### DISCUSSION THREAD

**Discussion by caraiz** [airsdk/Adobe-Runtime-Support#603] [Q&A]
## Problem Description

While the splash screen is shown, the app is running, so it can affect the app contents with behaviour depending on the time (custom splash, start animation, etc...)

Is it possible known when the splash have dissapeared or how much time the splash is at screen?

**Comment by BMGMobile** [comment]
We would also like to know how to better control this behavior on iOS. What triggers the hiding of the iOS splash screen? Is there a way to prolong the duration until we have content ready to be displayed in our game?
